php/Dockerfile-5.6

```diff
@@ -24,6 +24,8 @@ RUN apt-get update -q -y \
         libicu52 \
         libxslt1.1 \
         libmemcachedutil2 \
+# Required by eZ Commerce (added here to keep images in sync, feature wise)
```

Comment:
eZ Commerce do not run on php 5.6....

Reply:
So these are my changes, as I'd like images to have the same feature set and be simple to keep in sync. hence the `(added here to keep images in sync, feature wise)` to answer comments like:
> eZ Commerce do not run on php 5.6....

If you think it's still wrong we can drop it, but..

Reply:
Ok. I don't mind having it there then if it simplifies things
My point was just that eZ Platform 2.3 [requires php 7.1 or newer](https://github.com/ezsystems/ezplatform/blob/2.3/composer.json#L27)

Reply:
Yes I know, but these images are also for 1.x, and would like to:
- simplify maintaining them (so keeping them as much as possible in sync)
- have similar feature set / behaviour when moving form one php version to the other, e.g. platform can use imagemagic also, it's just that commerce has hardcoded this and for now does not support GD.